.github/workflows/build-packages-self-hosted.yml

```
@@ -3,28 +3,56 @@
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository with submodules
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
 
-      - name: Validate release version format
+      - name: Set build type
+        id: set-build-type
+        run: | 
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:60: Double quote to prevent globbing and word splitting

(shellcheck)